src/samplereader/SampleReader.h

```diff
@@ -14,6 +14,20 @@
 #include <kodi/AddonBase.h>
 #include <kodi/addon-instance/Inputstream.h>
 
+enum class CryptoMode : uint16_t
```

Comment:
please add a comment above this enum with something similar to this:
`// The crypto modes values must match android MediaCodec values`
We should start documenting the code better, this will make life easier for us and for future developers